src.ts/providers/json-rpc-provider.ts

```diff
@@ -76,7 +72,15 @@ export class JsonRpcProvider extends Provider {
         const response = await fetchJson(this.connection, JSON.stringify(request));
         if (response.error) {
             if (typeof response.error.data === 'object') {
-                throw new TypedError(response.error.data.error_message, response.error.data.error_type);
+                if (typeof response.error.data.error_message === 'string' && typeof response.error.data.error_type === 'string') {
```

Comment:
let's add a TODO with GH issue link to track removal of temporary stuff (not a blocker for merge)

Reply:
https://github.com/nearprotocol/nearlib/issues/173